tst/EnergyPlus/unit/ThermalComfort.unit.cc

```diff
@@ -948,3 +948,527 @@ TEST_F(EnergyPlusFixture, ThermalComfort_CalcIfSetPointMetWithCutoutTest)
+        
+        "BuildingSurface:Detailed,",
+        " SB12x10,                  !- Name",
+        " WALL,                     !- Surface Type",
+        " INT-WALL-1,               !- Construction Name",
+        " Spacex10,                 !- Zone Name",
+        " Adiabatic,                !- Outside Boundary Condition",
+        " ,                         !- Outside Boundary Condition Object",
+        " NoSun,                    !- Sun Exposure",
+        " NoWind,                   !- Wind Exposure",
+        " 0.0,                      !- View Factor to Ground",
+        " 4,                        !- Number of Vertices",
+        " 30.5, 0.0, 2.4,           !- X, Y, Z == > Vertex 1 {m}",
+        " 30.5, 0.0, 0.0,           !- X, Y, Z == > Vertex 2 {m}",
+        " 26.8, 3.7, 0.0,           !- X, Y, Z == > Vertex 3 {m}",
+        " 26.8, 3.7, 2.4;           !- X, Y, Z == > Vertex 4 {m}",
+        
+        "BuildingSurface:Detailed,",
+        " SB14x10,                  !- Name",
+        " WALL,                     !- Surface Type",
+        " INT-WALL-1,               !- Construction Name",
+        " Spacex10,                 !- Zone Name",
+        " Adiabatic,                !- Outside Boundary Condition",
+        " ,                         !- Outside Boundary Condition Object",
+        " NoSun,                    !- Sun Exposure",
+        " NoWind,                   !- Wind Exposure",
+        " 0.0,                      !- View Factor to Ground",
+        " 4,                        !- Number of Vertices",
+        " 3.7, 3.7, 2.4,            !- X, Y, Z == > Vertex 1 {m}",
+        " 3.7, 3.7, 0.0,            !- X, Y, Z == > Vertex 2 {m}",
+        " 0.0, 0.0, 0.0,            !- X, Y, Z == > Vertex 3 {m}",
+        " 0.0, 0.0, 2.4;            !- X, Y, Z == > Vertex 4 {m}",
+        
+        "BuildingSurface:Detailed,",
+        " SB15x10,                  !- Name",
+        " WALL,                     !- Surface Type",
+        " INT-WALL-1,               !- Construction Name",
+        " Spacex10,                 !- Zone Name",
+        " Adiabatic,                !- Outside Boundary Condition",
+        " ,                         !- Outside Boundary Condition Object",
+        " NoSun,                    !- Sun Exposure",
+        " NoWind,                   !- Wind Exposure",
+        " 0.0,                      !- View Factor to Ground",
+        " 4,                        !- Number of Vertices",
+        " 26.8, 3.7, 2.4,           !- X, Y, Z == > Vertex 1 {m}",
+        " 26.8, 3.7, 0.0,           !- X, Y, Z == > Vertex 2 {m}",
+        " 3.7, 3.7, 0.0,            !- X, Y, Z == > Vertex 3 {m}",
+        " 3.7, 3.7, 2.4;            !- X, Y, Z == > Vertex 4 {m}",
+        
+        "Output:Table:SummaryReports,",
+        "  AllSummary; !- Report 1 Name",
+        " ",
+        
+    });
+    
+    ASSERT_TRUE(process_idf(idf_objects));
+    
+    OutputProcessor::TimeValue.allocate(2);
```

Comment:
And when you merge in develop, this line will fail. Simply delete it.

Reply:
Will delete it.